indexing-lambda/src/test/java/edu/nix/AppTest.java

```diff
@@ -0,0 +1,38 @@
+package edu.nix;
+
+import junit.framework.Test;
+import junit.framework.TestCase;
+import junit.framework.TestSuite;
+
+/**
+ * Unit test for simple App.
+ */
+public class AppTest 
```

Comment:
Try not to include irrelevant or stub stuff to the PR

Reply:
Ok.